doc/user-manual/tools/command-line-options.rst

```diff
@@ -757,6 +757,37 @@ Pattern matching and equality
 
      Default: ``--infer-absurd-clauses``.
 
+.. option:: --forced-argument-recursion
+
+     .. versionadded:: 2.6.4
+
+     Allow the use of forced constructor arguments as termination metrics. This is inconsistent when combined with :option:`--large-indices`.
+     Since forced arguments are fixed by the constructor's type, when the K rule is enabled, they are "not really there". Thus, they do not provide enough justification to assert a recursive call is size-decreasing.
+
+     Forced argument recursion is disabled by the default when the K rule is enabled, and are enabled by default if :option:`--without-K` is given.
+
+.. option:: --no-forced-argument-recursion
+
+     .. versionadded:: 2.6.4
+
+     Opposite of :option:`--forced-argument-recursion`.
```

Comment:
Since both variants are added in the same version, I'd say we need no extra section for the "no" variant but have simply:
```
.. option:: --forced-argument-recursion, --no-forced-argument-recursion
```